.github/workflows/update-queue-times.yml

```diff
@@ -11,9 +11,17 @@ defaults:
 jobs:
   update-queue-rockset:
     runs-on: ubuntu-20.04
+    permissions:
+      id-token: write
     steps:
       - uses: actions/checkout@v3
       - run: yarn install --frozen-lockfile
+      - name: configure aws credentials
+        id: aws_creds
+        uses: aws-actions/configure-aws-credentials@v1.7.0
```

Comment:
Why 1.7.0 version?

Reply:
Good catch, we couldn't use v4 because of Node 20, but v3 works fine https://github.com/pytorch/test-infra/pull/4869